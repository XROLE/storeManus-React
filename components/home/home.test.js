import React from 'react';
import { shallow } from 'enzyme';
import Home from './home';

describe('Header', () => {
  it('Should render just the homepage and footer', () => {
    const component = shallow(
      <Home />,
    );
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
