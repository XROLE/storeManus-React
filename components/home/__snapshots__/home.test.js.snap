// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render just the homepage and footer 1`] = `
<div
  className="homee"
>
  <div
    className="logDiv"
  >
    <p>
       
      <i
        className="fas free fa-stream"
      />
         
      <span>
        Store Manus
      </span>
    </p>
  </div>
  <div
    className="descriptiveDiv"
  >
    <h1
      className="description"
    >
      Reliable Store Manager
    </h1>
    <a
      className="get-started-button"
      href="./views/signin.html"
    >
       
      <i
        className="far fa-hand-pointer"
      />
         Get Started
    </a>
    <p
      className="description2"
    >
      simple to use
    </p>
  </div>
</div>
`;
